ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -93,7 +93,7 @@ def _get_clip_inputs(draw):
         available_dtypes=helpers.get_dtypes("valid", prune_function=False)
     ).filter(lambda x: "bfloat16" not in x[0]),
 )
-def test_paddle_instance_property_device(
+def test_paddle_property_device(
```

Comment:
"property" should not be included in the test names.

Reply:
Done